containers/server-saline-image/run_saline.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [[ ! -z "$TZ" ]]; then
+    timedatectl set-timezone $TZ
+fi
+
+cp -r /etc/saline.defaults/saline* /etc/salt/
+
+mkdir -p /run/salt/master
+chown -R salt:salt /run/salt
+
+mkdir -p /etc/salt/pki/saline
+
+if [[ "$NOSSL" == "YES" ]]; then
+    cat > /etc/salt/saline.d/restapi.conf <<EOL
+restapi:
+  host: 0.0.0.0
+  disable_ssl: true
+EOL
+else
+    cp /etc/pki/tls/certs/spacewalk.crt /etc/salt/pki/saline/saline.crt
+    cp /etc/pki/tls/private/spacewalk.key /etc/salt/pki/saline/saline.key
+    chown -R salt:salt /etc/salt/pki/saline
+    chmod 0750 /etc/salt/pki/saline
+    cat > /etc/salt/saline.d/restapi.conf <<EOL
```

Comment:
It's a lot of `cp`, `chown`, `chmod`. Can you do it in the container at once and here only generate file based on `NOSSL` value?

Reply:
These files are from the shared volume and runtime ones, no any changes to do it during the build time of the container.

Reply:
`chmod 0750` here is not recursive and sets the permission to the directory only while the `chown` call is recursive to set salt owner on the certificate and key file, I can rearrange the orders of the calls, but without 0750 on the directory the permisison is wrong there.

Reply:
Ok, I was thinking directory permission is not important, but maybe it's true as you can? delete folder with certs having permissions correctly set

Reply:
Slightly changed the approach used in the script here to prevent extra copy of the certificate in the config directory, so it copies the certificate to runtime directory, but keep maintaining the correct permissions.